code/index.html

```diff
@@ -1,22 +1,134 @@
+</head>
+
+<body>
+<!-- Header with logo and nav -->
+  <header>
+    <img class="logoIcon" src="./cat.logo2.svg">
+    <nav>
+      <a href="">DESIGN</a>
+      <a href="">PHOTOGRPHY</a>
+      <a href="">FILM</a>
+      <a href="">ILLUSTRATION</a>
+    </nav>
+    <div class="hamburger">
+      <div></div>
+      <div></div>
+      <div></div>
+    </div>
+  </header>
+
+  <!-- Start picture -->
+  <section class="start-picture">
+    <img class="todaysPic" src="./3329759-01.jpg">
+    <div class="ref">
+      <h1>Design by Freepik</h1>
+    </div>
+  </section>
+
+  <!-- Big News  -->
+  <article class="big-arc">
+    <h4>Stay cool</h4>
+    <p>Lorem Ipsum</p>
+    <img src="./chole.gif" alt="">
+  </article>
+
+ <!-- Small News Section  -->
+  <section class="overview">
+    <section class="start-text">
+      <h3 class="date">28/8</h3>
+      <h2 class="autor"><strong>Todays<br>Illustrator:</strong><br>Bäm Stugendal</h2>
     </section>
-  </body>
-</html>
+
+    <article class="arc">
+      <img src="./rangnarak.jpg">
+      <h4>Ragnarök</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./Miss_Fireworks_Ani.gif">
+      <h4>Miss Fireworks</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./A_girl.jpg">
+      <h4>Chilling</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./plant_cat2.jpg">
+      <h4>Plant Cat</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./Mr_Bubble_Ani.gif">
+      <h4>Mr Bubbles</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./a_girl3.jpg">
+      <h4>Good Hair day</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./fuse.jpg">
+      <h4>Fuse</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./cats.jpg">
+      <h4>Liquid cats</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./a_girl2.jpg">
+      <h4>Aqua</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./house.jpg">
+      <h4>House</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./flying_cat.jpg">
+      <h4>Rocket Cat</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+
+    <article class="arc">
+      <img src="./frukost1.jpg">
+      <h4>Breakfest</h4>
+      <p>Lorem Ipsum</p>
+    </article>
+  </section>
+
+ <!-- Footer  -->
+  <footer>
+    <p>Made by Bäm Stugendal</p>
+    <div class="logo-social">
+      <a href="https://www.instagram.com/illustration.bam/"><img src="./Instagram_2.svg">
+      <a href="https://github.com/BamStugendal?tab=repositories"><img src="./Github_2.svg">
+  </div>
```

Comment:
This look so good on the website and the social media icons are a very nice touch! A little thing that is missing is the closing tag for the links. Just update this section with </a> after each image and you're good to go.